samples/subsys/video/capture_to_lvgl/README.rst

```diff
@@ -0,0 +1,82 @@
+.. zephyr:code-sample:: video-capture-to-lvgl
+   :name: Video capture to LVGL
+   :relevant-api: video_interface
+
+   This sample application demonstrates how to use the video API to retrieve video
+   frames from a capture device and display them on an LCD via the LVGL library.
+
+Description
+***********
+
+The application uses the :ref:`Video API <video_api>` to retrieve video frames from
+a video capture device, writes a frame count message to the console, and then sends
+the frame to an LCD display.
+
+Requirements
+************
+
+This sample requires a video capture device (e.g., a camera) and an LCD display.
+
+- :ref:`mini_stm32h743`
+- OV2640 camera module
+- LCD display
+
+Wiring
+******
+
+On the `WeAct Studio STM32H743`_, connect the OV2640 camera module and the 0.96" ST7735
+TFT LCD display. Connect a USB cable from a host to the micro USB-C connector on the
+board to receive console output messages.
+
+Building and Running
+********************
+
+For :ref:`mini_stm32h743`, build this sample application with the following commands:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/video/capture_to_lvgl/
+   :board: mini_stm32h743
+   :goals: build flash
+   :gen-args: -DCONFIG_BOOT_DELAY=5000
```

Comment:
really needed?

Reply:
Yes. Since the board uses USB CDC to print the log messages, some messages are missing when the delay is removed, but I can reduce it from 5000 to 2000.

Reply:
Changed to 2000